concourse/pipelines/concourse-lite-self-terminate.yml

```diff
@@ -0,0 +1,31 @@
+resources:
+  - name: delete-timer
+    type: time
+    source:
+      start: 20:00 -0000
+      stop: 21:00 -0000
```

Comment:
The autodelete-cloudfoundry will be running every 2h until the morning, trying to delete the deployment.

But this one we run it once because once is run, the machine is gone.  

But on the other hand, one can argue that maybe the job fails because some reason (docker registry down, AWS timeout, etc...) in which case it would make sense to retry every 1-2h.... If you want, I can make it the same.